sonic-net/sonic-buildimage
# Clear FABRIC_PORT_TABLE when swss is got down

#### Why I did it
fix #18554

Issue 18554(https://github.com/sonic-net/sonic-buildimage/issues/18554) describes an issue of that when stopping swss services on SFMs, the related fabric reachability information do not get remove from the database.

It's found that corresponding fabric port status information is in STATE_DB with key FABRIC_PORT_TABLE|PORTxxx.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Add clean table action for FABRIC_PORT_TABLE in STATE_DB when swss service is down.

#### How to verify it
Made changes and loaded the changes into chassis. Run "show fabric reachability -n asic7" to see the fabric ports. Also checked STATE_DB by using "sonic-db-cli -n asic7 STATE_DB hgetall "FABRIC_PORT_TABLE|PORT112" where port 112 is connected and up.

Issue "sudo systemctl stop swss@7". Check "show fabric reachability -n asic7". It's empty now.
Issue "sudo systemctl start swss@7" or "docker start swss7". Wait for 1 minutes. Run "show fabric reachability -n asic7", all connected port are displayed there. Also checked STATE_DB and all entries with key FABRIC_PORT_TABLE|PORTxxx are correct.
Issue "docker kill swss7". Repeat testing steps in 1). It works as exepcted.
Enter container swss7. Run "kill 35" where 35 is task id for orchagent. Watched container swss7 is stopped and then restarted. Watched the "Cleared FABRIC_PORT_TABLE from STATE_DB for swss7" log showed up after container is stopped.
#### Which release branch to backport (provide reason below if selected)

#### Tested branch (Please provide the tested image version)
master, 202205
